xmtp_mls/src/groups/device_sync.rs

```diff
@@ -163,54 +148,7 @@ pub struct SyncWorker<ApiClient, V> {
 
     // Number of events processed
     #[cfg(any(test, feature = "test-utils"))]
-    handle: Arc<WorkerHandle>,
-}
-
-#[cfg(any(test, feature = "test-utils"))]
-pub struct WorkerHandle {
```

Comment:
this was giving a bunch of `unused` in `bindings_node`, since it doesn't rely on `test-utils` in their dev dependencies. I moved it to `utils/test/mod.rs` to make it easier to manage the dependencies specific to the WorkerHandle and those in non-test stuff